milestone2.ts

```diff
@@ -93,23 +101,37 @@ let findAverage: GetNumber; // Define lambda function here
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+
+let findIndexAns: GetNumber = (data) => {
+  let first = data.find(person => person.city === "SF");
+  if (first === undefined)
+    return -1;
+  return data.indexOf(first);
```

Comment:
you can use the findIndex function here instead, which will return the index of the first element meeting those conditions, or -1 if it doesn't find anything